dlpy/tests/test_applications.py

```diff
@@ -853,3 +853,63 @@ def test_unet_with_bn(self):
         self.assertTrue(len(model.layers) == 33 + 9 * 2)
         self.assertTrue(model.layers[12].output_size == (256, 256, 256))
         model.print_summary()
+
+    def test_mobilenetv2_onnx_1(self):
+        from dlpy.applications import MobileNetV2_ONNX
+
+        if self.data_dir is None:
+            unittest.TestCase.skipTest(self, "DLPY_DATA_DIR is not set in the environment variables")
+        file_dependency = self.data_dir + 'MobileNetV2_ONNX.sashdat'
```

Comment:
This needs to be on the local data dir not in data_dir. Right? you need the onnx file to be on the client side rather than on the server side?

Reply:
It accepts a sashdat file that should be accessed on server side.

Reply:
cool